test/swoosh/adapters/oh_my_smtp_test.exs

```diff
@@ -1,8 +1,8 @@
-defmodule Swoosh.Adapters.OhMySmtpTest do
+defmodule Swoosh.Adapters.MailPaceTest do
```

Comment:
Looks like the file name should match the module name

Reply:
haha yep.